msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -537,7 +537,6 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 			ApplicationName="$(_AppBundleName)"
 			MainAssembly="$(OutputPath)$(TargetName)$(TargetExt)"
 			TargetFrameworkMoniker="$(_ComputedTargetFrameworkMoniker)"
-			UseXamMacFullFramework="$(UseXamMacFullFramework)"
```

Comment:
was the `UseXamMacFullFramework` removal intentional ?
it's not mentioned in the commit message and this is specific to XM